java/greeter/docker-compose.yml

```diff
@@ -35,51 +35,12 @@ services:
   ###############################################################
 
   statefun:
-    image: apache/flink-statefun-playground:3.2.0
+    image: apache/flink-statefun-playground:3.2.0-1.0
```

Comment:
I don't think it is needed, you can republish the same docker image (`apache/flink-statefun-playground:3.2.0`)

Reply:
Oh really? So it would only be a problem if it is cached somewhere, right?